### PR TITLE
docs(zh): translate Developer Guide and related docs into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
**What type of PR is this?**
Translate Developer Guide and related docs into Chinese:
```
site/content/zh/contributions/DEVELOP.md
site/content/zh/latest/tasks/security/jwt-authentication.md
site/content/zh/latest/tasks/traffic/grpc-routing.md
```

### CODE REVIEW

File: site/content/zh/contributions/DEVELOP.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+* 运行 `make build BINS="envoy-gateway"` 来构建 Envoy Gateway 二进制文件。
+* 运行 `make build BINS="egctl"` 来构建 egctl 二进制文件。
+
+**注意：**二进制文件在 `bin/$OS/$ARCH` 目录中生成，例如，`bin/linux/amd64/`。
+
+### 测试 {#testing}
+
+* 运行 `make test` 来运行 golang 测试。
+
+* 运行 `make testdata` 生成黄金 YAML 测试数据文件。
```
Comment (shawnh2) [inline site/content/zh/contributions/DEVELOP.md:49]:
```suggestion
* 运行 `make testdata` 生成标准的 YAML 测试数据文件。
```

**Review by shawnh2 — APPROVED** [review]
Looking Good! Thanks.

File: site/content/zh/latest/tasks/security/jwt-authentication.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+```shell
+kubectl delete securitypolicy/jwt-example
+```
+
+## 后续步骤 {#next-steps}
+
+查看[开发者指南](../../../contributions/develop)参与该项目。
+
+[SecurityPolicy]: ../../../contributions/design/security-policy
```
[the remark this answers is not in the document]
Comment (wilsonwu) [reply site/content/zh/latest/tasks/security/jwt-authentication.md:159]: Aha, thanks, sorry for miss this, let me translate it ASAP.

[the remark this answers is not in the document]
Comment (wilsonwu) [reply site/content/zh/latest/tasks/security/jwt-authentication.md:159]: Done

File: site/content/zh/contributions/design/security-policy.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+## 实现 {#implementation}
+
+`SecurityPolicy` 是一个[策略附件][]类型的 API，可用于扩展 [Gateway API][] 来定义身份验证和鉴权规则。
+
+### 示例 {#example}
+
+以下示例重点介绍了用户如何配置此 API。
+
+```
```
Comment (shawnh2) [inline site/content/zh/contributions/design/security-policy.md:25]:
```suggestion
```yaml
```

**Review by shawnh2 — APPROVED** [review]
one nit, LGTM! Thanks!

**Review by zhaohuabing — APPROVED** [review]
LGTM thanks!

**Pull request merged**